docs/building-a-form/available-components.md

```diff
@@ -182,9 +207,45 @@ A group of options where the user can select multiple items.
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+Each individual checkbox is used to store `boolean` data. If you want to include a group of checkboxes, you would include separate fields for each checkbox, with `type: 'boolean'` passed to the `schema`.
```

Comment:
> If you want to include a group of checkboxes, you would include separate fields for each checkbox, with `type: 'boolean'` passed to the `schema`.

A little tighter: "To include a group of checkboxes, include separate fields for each checkbox, with `type: 'boolean'` passed to the `schema`."